packages/addons/service/jellyfin/package.mk

```
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="jellyfin"
+PKG_VERSION="bootstrap"
+PKG_REV="100"
+PKG_ARCH="any"
+PKG_LICENSE="GPLv2"
+PKG_SITE="https://jellyfin.org/"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="service"
+PKG_SHORTDESC="Jellyfin: The Free Software Media System"
+PKG_LONGDESC="Jellyfin is the volunteer-built media solution that puts you in control of your media. Stream to any device from your own server, with no strings attached. Your media, your server, your way."
```

```suggestion
PKG_LONGDESC="Jellyfin is a Free Software Media System that puts you in control of managing and streaming your media. Stream to any device from your own server. Your media, your server, your way."
```

less the marketing blahblah :)